src/host/run.cpp

```diff
@@ -1021,7 +1021,7 @@ namespace ccf
     constexpr size_t max_close_iterations = 1000;
     int close_iterations = max_close_iterations;
     int loop_close_rc = 0;
-    while (close_iterations > 0)
+    while (close_iterations-- > 0)
```

Comment:
I don't think we should decrement during the comparison? Results in `close_iterations` ending up at `-1`, and we want to still use it in the later log line? Also not sure why this has changed to `int` rather than `size_t`.